source/fundamentals/csfle.txt

```diff
@@ -0,0 +1,284 @@
+==================================
+Client-Side Field Level Encryption
+==================================
+
+.. default-domain:: mongodb
+
+Introduced in MongoDB 4.2, Client-Side Field Level Encryption (CSFLE) allows administrators and developers to encrypt
+**specific** data fields and still leverage other MongoDB encryption features.
+
+With CSFLE, developers encrypt fields client-side without any server-side configuration or directives. CSFLE supports
+workloads where applications must guarantee that unauthorized parties, including server administrators, cannot read the
+encrypted data.
+
+The following is a quick introduction to CSFLE using the Java driver. For more in-depth information, see the
+:manual:`CSFLE reference-documentation </core/security-client-side-encryption/>` and the
```

Comment:
Yep, just a typo that didn't get picked up by my spelling/grammar addon (probably because of the `-`). I just chose the name reference documentation, we could just as easily call it anything else.